File: iOS/ssabree/ssabree/Core/Models/UserInfo.swift
import Foundation

struct UserInfo {
    let userId: String
    var password: String
    var nickname: String
    var role: String
    var uid: Int

    init(userId: String) {
        self.userId = userId
        self.password = ""
        self.nickname = ""
        self.role = ""
        self.uid = -1
    }

    init(userId: String, password: String) {
        self.userId = userId
        self.password = password
        self.nickname = ""
        self.role = ""
        self.uid = -1
    }

    init(userId: String, password: String, nickname: String, role: String, uid: Int) {
        self.userId = userId
        self.password = password
        self.nickname = nickname
        self.role = role
        self.uid = uid
    }
}

File: android/ssabree/app/src/main/java/com/ssafy/ssabree/core/utils/model/UserInfo.kt
package com.ssafy.ssabree.core.utils.model

class UserInfo(val userId: String) {

    var pw: String = ""
    var nickname: String = ""
    var role: String = ""

    var uId: Int = -1


    constructor(id: String, pw:String):this(id) {
        this.pw = pw
    }

    constructor( id: String, pw:String,nickname:String, role: String, uId: Int): this(id) {
        this.pw = pw
        this.nickname = nickname
        this.role = role
        this.uId = uId
    }
}